keras_cv/layers/serialization_test.py

```diff
@@ -36,54 +36,96 @@ def config_equals(config1, config2):
 
 class SerializationTest(tf.test.TestCase, parameterized.TestCase):
     @parameterized.named_parameters(
-        ("AutoContrast", preprocessing.AutoContrast, {"value_range": (0, 255)}),
-        ("ChannelShuffle", preprocessing.ChannelShuffle, {"seed": 1}),
-        ("CutMix", preprocessing.CutMix, {"seed": 1}),
-        ("Equalization", preprocessing.Equalization, {"value_range": (0, 255)}),
```

Comment:
Why are these lines reformatted? 🤔

@LukeWood

Reply:
@AdityaKane2001 why would these be reformatted?

Reply:
I'm not really sure. I'll run the format script again.

Reply:
any luck